pull/github.go

```diff
@@ -225,55 +175,146 @@ func (ghc *GitHubContext) Branches() (base string, head string, err error) {
 }
 
 func (ghc *GitHubContext) loadTimeline() error {
-	var opts github.ListOptions
-	var allEvents []*pullrequestEvent
+	var q struct {
+		Repository struct {
+			PullRequest struct {
+				Timeline struct {
+					PageInfo struct {
+						EndCursor   githubv4.String
+						HasNextPage bool
+					}
+					Nodes []*timelineEvent
+				} `graphql:"timeline(first: 100, after: $cursor)"`
```

Comment:
Where does this `100` come from?

Reply:
It's an arbitrary page size, but 100 is the [maximum allowed by GitHub](https://developer.github.com/v4/guides/resource-limitations/#node-limit).